runtime/cudaq/platform/pasqal/PasqalServerHelper.cpp

```diff
@@ -65,13 +68,39 @@ PasqalServerHelper::createJob(std::vector<KernelExecution> &circuitCodes) {
 
     tasks.push_back(taskRequest);
   }
-  cudaq::info("Created job payload for Pasqal, targeting device {}", backendConfig.at("machine"));
+
+  cudaq::info("Created job payload for Pasqal, targeting device {}",
+              backendConfig.at("machine"));
+  
   // Return a tuple containing the job path, headers, and the job message
-  const std::string baseUrl = "https://pasqal.cloud";
-  return std::make_tuple(baseUrl + "v1/batches", getHeaders(), tasks);
+  return std::make_tuple(baseUrl + "v1/" + backendConfig.at("server"), getHeaders(), tasks);
```

Comment:
Sorry, this is not actually what I meant (giving urls in speech is hard :p )
Thinking about it though we're too far away from knowing the exact future needs here so I suggest we go back to
```suggestion
  return std::make_tuple(baseUrl + "/v1/cuda-q/job") + , getHeaders(), tasks);
```
where for baseUrl I expect `https://apis.pasqal.cloud` (remove trailing `/`)